PyViCare/PyViCareHeatingDevice.py

```diff
@@ -196,6 +196,11 @@ def setDomesticHotWaterTemperature2(self, temperature):
         return self.service.setProperty("heating.dhw.temperature.temp2", "setTargetTemperature",
                                         {"temperature": int(temperature)})
 
+    @handleAPICommandErrors
+    def setDhwOperatingMode(self, mode):
```

Comment:
```suggestion
    def setDomesticHotWaterOperatingMode(self, mode):
```

Reply:
Could you please change the method name to be consistent with the other method names?